src/main/resources/templates/admin/organization.html

```diff
@@ -363,8 +363,8 @@ <h4>Assign New Terraformation Contact (interim solution - until we add TW suppor
     <p>
         The current Terraformation Contact will be removed from the org (if one exists) and the new one will be assigned.
     </p>
-    <label for="email" style="margin-top: 20px">Enter new Terraformation Contact email:</label>
-    <input type="email" id="terraformationContactEmail" name="terraformationContactEmail" style="min-width: 300px; padding: 2px;">
+    <label for="email" style="margin-top: 20px">Enter the Terraformation Contact's <i>(@terraformation.com)</i> email:</label>
+    <input type="email" id="terraformationContactEmail" name="terraformationContactEmail" style="min-width: 300px; padding: 2px;" pattern="^\S+@terraformation\.com$">
```

Comment:
Cool. Does this render the form non-submittable when the pattern isn't matched?

Reply:
Yes, the form will pop-up a message above the input asking to correct the pattern.